include/sdf/Root.hh

```diff
@@ -183,6 +198,13 @@ namespace sdf
     /// an error code and message. An empty vector indicates no error.
     public: Errors UpdateGraphs();
 
+    /// \brief Create and return an SDF element filled with data from this
+    /// root.
+    /// \param[in] _useIncludeTag This will pass the _useIncludeTag to
+    /// sdf::Model::ToElement.
+    /// \return SDF element pointer with updated root values.
+    public: sdf::ElementPtr ToElement(bool _useIncludeTag = true) const;
```

Comment:
I have a feeling this will need more parameters at some point. Something like the `sdf::PrintConfig` class or an enum would work, but we can leave it for future work if you don't have time for it now.

Reply:
A separate class might be needed, but I'd rather add the functionality when I have concrete use case.